modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -165,6 +166,9 @@ const setup = (editor: Editor): RenderInfo => {
       onEscape: () => {
         editor.focus();
       },
+      onToolbarToggled: (state: boolean) => {
+        fireToggleToolbarDrawer(editor, state);
```

Comment:
This is almost there, but still doesn't follow the TinyMCE style guide sorry. You should be using `import * as Events` and `Events.fireToggleToolbarDrawer` here so that it improves readability and makes it clear it's not a local function.

Reply:
@lnewson The issue is that there is already an `Events` variable in this scope, which setups document and other events. L456

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8238/commits/41f53bd6542c55a41658022a49469695e27efdb5).